### ISSUE TITLE
Crash Report: same with #827

### ISSUE THREAD

**Issue by GeoDaTester** [GeoDaCenter/geoda#820]
Details:
os: 1-10-10
vs: 1-8-16-25
 GdaFrame::ShowOpenDatasourceDlg()
ConnectDatasourceDlg::InitSamplePanel()
Check auto update:
AutoUpdate::CheckUpdate()
AutoUpdate::GetCheckList()
/Users/Julia/Box Sync/Julia/CSDS/Data/SSI_Welfare Data Sandy Wong/2010/contig2010_var2010.shp
Entering ConnectDatasourceDlg::SaveRecentDataSource
Exiting ConnectDatasourceDlg::SaveRecentDataSource
Entering Project::Project (new project)
Project::CommonProjectInit()
Entering Project::InitFromOgrLayer
Datasource name:/Users/Julia/Box Sync/Julia/CSDS/Data/SSI_Welfare Data Sandy Wong/2010/contig2010_var2010.shp
Project::SetupEncoding()
UTF-8
Project::GetSpatialReference()
Exiting Project::Project
Click GdaFrame::InitWithProject()
Open TableFrame.
Open MapFrame.
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetCanvasTitle()
MapCanvas::GetCanvasTitle()
MapCanvas::GetCanvasTitle()
Click GdaFrame::OnToolsWeightsManager
Entering WeightsManFrame::WeightsManFrame
In WeightsManFrame::SetDetailsForItem
In WeightsManFrame::OnActivate
Exiting WeightsManFrame::WeightsManFrame
In WeightsManFrame::OnLoadBtn
In WeightsManFrame::OnActivate
In WeightsManFrame::OnActivate
In WeightsManFrame::update(WeightsManState* o)
In WeightsManFrame::update(WeightsManState* o)
In WeightsManFrame::OnWListItemSelect
In WeightsManFrame::SetDetailsForItem
In WeightsManFrame::SetDetailsForItem
In WeightsManFrame::OnActivate
Open GetisOrdMapFrame (OnOpenGetisOrd).
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
SSI10
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
Open GetisOrdMapFrame (OnOpenGetisOrd).
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
TANF10
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
Open GetisOrdMapFrame (OnOpenGetisOrd).
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
TANF00
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
Open GetisOrdMapFrame (OnOpenGetisOrd).
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
SSI00
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::AddTimeVariantOptionsToMenu()
In GdaFrame::OnShowAsConditionalMap()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
SSI00
TANF00
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetNameWithTime()
Open ConditionalNewFrame.
In ConditionalClusterMapFrame::OnActivate()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::AddTimeVariantOptionsToMenu()
In GdaFrame::OnShowAsConditionalMap()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
TANF10
TANF00
MapCanvas::GetNameWithTime()
Open ConditionalNewFrame.
In ConditionalClusterMapFrame::OnActivate()
Open RegressionDlg.
RegressionDlg::InitVariableList()
RegressionDlg::OnCButton1Click()
RegressionDlg::OnCButton2Click()
RegressionDlg::OnCButton2Click()
RegressionDlg::OnCButton2Click()
Click RegressionDlg::OnRunClick
y:SSI10
x:
TANF10
TANF00
SSI00
RegressionDlg::DisplayRegression()
Open RegressionReportDlg.
In RegressionReportDlg::OnClose()
In ConditionalClusterMapFrame::OnActivate()
Close RegressionDlg.
RegressionDlg::OnCButton3Click()
Click RegressionDlg::OnRunClick
y:SSI10
x:
TANF10
TANF00
RegressionDlg::DisplayRegression()
Open RegressionReportDlg.
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
TANF10
SSI10
Open ScatterNewPlotFrame.
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
TANF00
SSI10
Open ScatterNewPlotFrame.
In TableFrame::OnActivate
Entering GdaFrame::OnMapChoices
In GdaFrame::OnOpenHinge15()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
LQ10
Open HistogramFrame.
In HistogramFrame::OnActivate()
In TableFrame::OnActivate
In ConditionalClusterMapFrame::OnActivate()
RegressionDlg::~RegressionDlg()
Close RegressionDlg.
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::AddTimeVariantOptionsToMenu()
In GdaFrame::OnShowAsConditionalMap()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
LQ10
LQ00
MapCanvas::GetNameWithTime()
Open ConditionalNewFrame.
In ConditionalClusterMapFrame::OnActivate()
Entering GdaFrame::OnMapChoices
In GdaFrame::OnOpenHinge15()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
LQ10
Open MapFrame.
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
Open TimeChooserDlg.
Open VarGroupingEditorDlg.
In VarGroupingEditorDlg::OnNewGroupNameChange
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnAddToListClick
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnAddToListClick
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnNewGroupNameChange
In VarGroupingEditorDlg::OnCreateGrpClick
In VarGroupingEditorDlg::OnNewGroupNameChange
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnAddToListClick
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnAddToListClick
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnNewGroupNameChange
In VarGroupingEditorDlg::OnIncludeListSelection
In VarGroupingEditorDlg::OnIncludeListEdit
In VarGroupingEditorDlg::OnIncludeListEditEnd
In VarGroupingEditorDlg::OnIncludeListEditEnd
In VarGroupingEditorDlg::OnIncludeListSelection
In VarGroupingEditorDlg::OnIncludeListSelection
In VarGroupingEditorDlg::OnClose
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
SSI10
Open BoxPlotCanvas.
Close BoxPlotCanvas.
Open VarGroupingEditorDlg.
In VarGroupingEditorDlg::OnNewGroupNameChange
In VarGroupingEditorDlg::OnClose
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
LQ
Open BoxPlotCanvas.
Close BoxPlotCanvas.
Open VarGroupingEditorDlg.
In VarGroupingEditorDlg::OnNewGroupNameChange
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnAddToListClick
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnClose
Open VarGroupingEditorDlg.
In VarGroupingEditorDlg::OnNewGroupNameChange
In ConditionalClusterMapFrame::OnActivate()
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnAddToListClick
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnAddToListClick
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnNewGroupNameChange
In VarGroupingEditorDlg::OnCreateGrpClick
In VarGroupingEditorDlg::OnNewGroupNameChange
In VarGroupingEditorDlg::OnUngroupedListSelection
In VarGroupingEditorDlg::OnClose
In ConditionalClusterMapFrame::OnActivate()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
SSI
Open BoxPlotCanvas.
Close BoxPlotCanvas.
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
SSI
SSI
Open CartogramNewFrame.
Project::GetCentroids(std::vector<double>& x, std::vector<double>& y)
Project::GetCentroids()
Project::GetVoronoiRookNeighborGal()
Project::GetVoronoiRookNeighborMap()
Project::IsPointDuplicates()
Project::GetCentroids(std::vector<double>& x, std::vector<double>& y)
Project::GetCentroids()
Project::GetCentroids(std::vector<double>& x, std::vector<double>& y)
Project::GetCentroids()
In CartogramNewFrame::OnActivate()
In CartogramNewFrame::OnActivate()
In CartogramNewFrame::OnActivate()
In TableFrame::OnActivate
In TableFrame::OnLabelLeftClickEvent()
In TableFrame::OnLabelLeftClickEvent()
In TableFrame::OnLabelLeftDClickEvent()
Open DataViewerAddColDlg.
DataViewerAddColDlg::OnEditName()
DataViewerAddColDlg::OnEditDecimals()
DataViewerAddColDlg::OnEditLength()
DataViewerAddColDlg::OnEditDecimals()
DataViewerAddColDlg::OnEditName()
DataViewerAddColDlg::OnEditName()
DataViewerAddColDlg::OnEditName()
DataViewerAddColDlg::OnEditName()
In DataViewerAddColDlg::OnChoiceType()
DataViewerAddColDlg::OnEditDecimals()
DataViewerAddColDlg::OnEditLength()
Entering DataViewerAddColDlg::OnOkClick
DataViewerAddColDlg::OnEditName()
DataViewerAddColDlg::OnEditName()
Entering DataViewerAddColDlg::OnOkClick
Inserting new column FIPSNR into Table
In TableFrame::OnActivate
Open FieldNewCalcSheetDlg.
In FieldNewCalcSheetDlg::OnPageChange()
Open DataViewerEditFieldPropertiesDlg
In DataViewerEditFieldPropertiesDlg::OnGridComboBox
1
Entering DataViewerEditFieldPropertiesDlg::OnFieldSelected
Entering DataViewerEditFieldPropertiesDlg::OnCellChanging
changing row 1, col 1 from string to integer
Click GdaFrame::OnSaveProject
Entering Project::SaveDataSourceData
Project::SaveOGRDataSource()
Entering Project::SaveDataSourceAs
New Datasource Name:/Users/Julia/Box Sync/Julia/CSDS/Data/SSI_Welfare Data Sandy Wong/2010/GdaTmp_contig2010_var2010.shp
Project::GetSpatialReference()
Project::GetGdaGeometries()
Exiting Project::SaveDataSourceData
Entering Project::SaveProjectConf
Project::GetProjectFullPath()
In Project::UpdateProjectConf
Exiting Project::SaveProjectConf
In TableFrame::OnActivate
Open MergeTableDlg.
Entering MergeTableDlg::OnOpenClick()
Entering ConnectDatasourceDlg::OnOkClick
ConnectDatasourceDlg::CreateDataSource()
Open Datasource:/Users/Julia/Box Sync/Julia/CSDS/Data/2016 election/County_Level_Election_Results_12-16-master/US_County_Level_Presidential_Results_12-16a.csv
Open Layer:US_County_Level_Presidential_Results_12-16a
Entering ConnectDatasourceDlg::SaveRecentDataSource
Exiting ConnectDatasourceDlg::SaveRecentDataSource
In TableFrame::OnActivate
Exiting ConnectDatasourceDlg::OnOkClick
ds:/Users/Julia/Box Sync/Julia/CSDS/Data/2016 election/County_Level_Election_Results_12-16-master/US_County_Level_Presidential_Results_12-16a.csv layerUS_County_Level_Presidential_Results_12-16a
In MergeTableDlg::OnKeyChoice()
In MergeTableDlg::OnKeyChoice()
Entering MergeTableDlg::OnIncAllClick()
In MergeTableDlg::OnMergeClick()
Open FieldNameCorrectionDlg:
Click FieldNameCorrectionDlg::OnOkClick
In TableFrame::OnActivate
In TableFrame::OnActivate
In ConditionalClusterMapFrame::OnActivate()
Entering GdaFrame::OnMapChoices
In GdaFrame::OnOpenHinge15()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
votes_2017
Open MapFrame.
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
Entering GdaFrame::OnMapChoices
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnLabelLeftClickEvent()
In TableFrame::OnActivate
In TableFrame::OnActivate
In TableFrame::OnLabelLeftClickEvent()
In TableFrame::OnActivate
In TableFrame::OnActivate
Click GdaFrame::OnSaveProject
Entering Project::SaveDataSourceData
Project::SaveOGRDataSource()
Entering Project::SaveDataSourceAs
New Datasource Name:/Users/Julia/Box Sync/Julia/CSDS/Data/SSI_Welfare Data Sandy Wong/2010/GdaTmp_contig2010_var2010.shp
Project::GetSpatialReference()
Project::GetGdaGeometries()
Exiting Project::SaveDataSourceData
Entering Project::SaveProjectConf
In Project::UpdateProjectConf
Exiting Project::SaveProjectConf
In TableFrame::OnActivate
MapCanvas::~MapCanvas()
Entering GdaFrame::OnMapChoices
In GdaFrame::OnOpenHinge15()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
per_g_2016
Open MapFrame.
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
per_d_2016
SSI
Open ScatterNewPlotFrame.
MapCanvas::AddTimeVariantOptionsToMenu()
In CartogramNewFrame::OnActivate()
In ConditionalClusterMapFrame::OnActivate()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::AddTimeVariantOptionsToMenu()
In GdaFrame::OnShowAsConditionalMap()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
per_d_2016
per_g_2016
MapCanvas::GetNameWithTime()
Open ConditionalNewFrame.
In ConditionalClusterMapFrame::OnActivate()
In TableFrame::OnActivate
In TableFrame::OnLabelLeftClickEvent()
In TableFrame::OnLabelLeftDClickEvent()
In TableFrame::OnLabelLeftClickEvent()
In TableFrame::OnLabelLeftDClickEvent()
Entering GdaFrame::OnMapChoices
In GdaFrame::OnOpenHinge15()
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
Stipend10
Open MapFrame.
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
In TableFrame::OnActivate
In GdaFrame::OnMoveSelectedToTop
Entering GdaFrame::OnMapChoices
Open GetisOrdMapFrame (OnOpenGetisOrd).
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
SSI
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
Open GetisOrdMapFrame (OnOpenGetisOrd).
Open VariableSettingsDlg
Click VariableSettingsDlg::OnOkClick:
LQ
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
VariableSettingsDlg::GetWeightsId() weight: contig2010_var2010_queen
MapCanvas::MapCanvas()
MapCanvas::ChangeMapType()
MapCanvas::VarInfoAttributeChange()
MapCanvas::CreateAndUpdateCategories()
MapCanvas::GetNameWithTime()
MapCanvas::AddTimeVariantOptionsToMenu()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
In TableFrame::OnActivate
Open ScatterPlotMatFrame.
In ScatterPlotMatFrame::OnActivate
MapCanvas::AddTimeVariantOptionsToMenu()
In ScatterPlotMatFrame::OnActivate
Open VarGroupingEditorDlg.
In VarGroupingEditorDlg::OnNewGroupNameChange
In TimeChooserDlg::OnPlayPauseButton
In TimeChooserDlg::ChangeTime
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
In TimeChooserDlg::ChangeTime
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
In TimeChooserDlg::ChangeTime
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
In TimeChooserDlg::ChangeTime
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
In TimeChooserDlg::OnPlayPauseButton
In ScatterPlotMatFrame::OnActivate
In TimeChooserDlg::OnPlayPauseButton
In TimeChooserDlg::ChangeTime
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
In TimeChooserDlg::ChangeTime
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
In TimeChooserDlg::ChangeTime
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::TimeChange()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetCanvasTitle()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
MapCanvas::GetNameWithTime()
In TimeChooserDlg::OnPlayPauseButton
Open ScatterPlotMatFrame.
In ScatterPlotMatFrame::OnActivate

**Comment by jkoschinsky** [comment]
This was me. Data: SSI (non-public).
Version: 1.8.16.25 Mac

I had a lot of views open: 

- several univariate local G
- conditional maps of this
- regular maps
- what crashed it was 2nd scatterplot matrix

**Comment by jkoschinsky** [comment]
may be related to crash?

<img width="423" alt="screen shot 2017-05-13 at 7 34 46 am" src="https://cloud.githubusercontent.com/assets/13679743/26025500/bde6b216-37ae-11e7-9fce-35e0786a68ae.png">

**Comment by YidanJ-Wu** [comment]
Version 1.8.16.25 Windows 8 
I tried making one scatterplot matrix, then two side by side. Did not crash. 
![image](https://cloud.githubusercontent.com/assets/22801047/26029684/489a1af4-3801-11e7-95c4-5de82493755b.png)

**Comment by YidanJ-Wu** [comment]
Version 1.8.16.25 Windows 8 
Ran the scatter plot matrix again, this time with a different file. 
![image](https://cloud.githubusercontent.com/assets/22801047/26029740/53266b48-3802-11e7-8889-158800c942a3.png)
No crash, but I did notice that the columns with foreign characters aren't showing up as options. Is that by design or a bug? 
![image](https://cloud.githubusercontent.com/assets/22801047/26029746/7ba2b054-3802-11e7-9462-48d716b92c69.png)
